tests/cross/src/test/scala/tests/pc/CompletionKeywordSuite.scala

```diff
@@ -379,7 +379,9 @@ class CompletionKeywordSuite extends BaseCompletionSuite {
       |  val x: Map[Int, new@@]
       |}
     """.stripMargin,
-    ""
+    "",
+    // to avoid newMain annotation
+    filter = str => !str.contains("newMain")
```

Comment:
Should the annotation even be offered here? Is that a bug in Dotty?

Reply:
It's a type, so I think it's valid in most places. The same works currently with for example `tailrec`. Maybe it can be improved for cases though, but it should automatically do `_:` instead I think